src/PackageWatcher.ts

```diff
@@ -14,7 +15,13 @@ export class PackageWatcher {
     private packageFileWatcher?: vscode.FileSystemWatcher;
     private resolvedFileWatcher?: vscode.FileSystemWatcher;
 
-    constructor(private workspaceRoot: string) { }
+    private ctx: Ctx
+    private workspaceRoot: string
+
+    constructor(workspaceRoot: string, ctx: Ctx) {
```

Comment:
This can just be:

```
constructor(
  private workspaceRoot: string
  private ctx: Ctx
) { }
```

This seems to be the common pattern in TypeScript.

Reply:
Done